gp-includes/misc.php

```diff
@@ -331,3 +331,40 @@ function gp_gmt_strtotime( $string ) {
 
 	return $time;
 }
+
+/**
+ * Determines the format to use based on the selected format type or by auto detection based on the file name.
+ *
+ * Used during import of translations and originals.
+ *
+ * @param string $selected_format The format that the user selected on the import page.
+ * @param string $filename The filname that was uploaded by the user.
+ * @return object A GP_Format child object.
```

Comment:
Should be `@return GP_Format|null A GP_Format child instance or null if format not found`.